frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java

```diff
@@ -581,6 +581,8 @@
 
     public static final String PREDEFINED_TAGS;
 
+    public static final String PROJECT_GROUPS = "projectGroups";
```

Comment:
Maybe the same variable can be reused, instead of creating a new one
`PortalConstants.ORGANIZATIONS`

Reply:
Used the same variable.